src/components/SearchFilters/appliedFilterUtils.ts

```diff
@@ -49,6 +49,7 @@ export const addLabelPropAndParseFilters = (
     // for applied filter fields (yet). This is mainly the unsupported holding
     // location filter (eg filters[holdingLocation][0]=loc:scff2), which is
     // used by devs to assist QA. See line 69 for explanation of date exclusion.
+    const ignoreFields = ["date", "holdingLocation"]
```

Comment:
Where is this being used?

Reply:
I ran into some weirdness that I think was actually some long cached code or browser response. I started to try to fix it, but when I restarted the server it cleared itself up. Will remove.